pandas/core/internals/blocks.py

```diff
@@ -450,6 +450,20 @@ def f(mask, val, idx):
 
         return self.split_and_operate(None, f, inplace)
 
+    def _split(self) -> List["Block"]:
+        """
+        Split a block into a list of single-column blocks.
+        """
+        assert self.ndim == 2
+
+        new_blocks = []
```

Comment:
might be worth adding some unit tests for this (e.g. on a block that is empty, since column and multilple)

Reply:
good idea, was accidentally making a copy instead of a view!

Reply:
wow, testing is good!